tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
add LICENSE

Reply:
Maybe there is a reason for that, but if not I know from kubevirt/kubevirt the it's recommended to add LICENSE and I did add them in all other linter implementations (hco, cdi, kubevirt)